files/es/learn/common_questions/how_does_the_internet_work/index.html

```diff
@@ -50,11 +50,11 @@ <h3 id="Una_simple_red">Una simple red</h3>
 
 <p><img alt="Dos computadoras conectadas entre sí" src="https://mdn.mozillademos.org/files/8441/internet-schema-1.png" style="height: 152px; width: 600px;"></p>
 
-<p>La red no se limita a dos ordenadores, se pueden conectar tantos como se deseen aunque siendo más complicado cada vez. Por ejemplo, para conectar diez ordenadores, se necesitarían 45 cables y unos nueve conectores por ordenador!</p>
+<p>Pero una red no se limita a dos ordenadores, se pueden conectar tantos como se desee; sin embargo, rápidamente se complejizará... Por ejemplo, para conectar diez ordenadores, se necesitarían ¡45 cables y unos nueve conectores por ordenador!</p>
```

Comment:
* Permanece "La" ya que se habla de la red del ejemplo
* Evitar el uso de puntos suspensivos, y en tiempos dar preferencia al infinitvo

```suggestion
<p>La red no se limita a dos ordenadores, se pueden conectar tantos como se desee; sin embargo, rápidamente se volverá más compleja. Por ejemplo, para conectar diez ordenadores, se necesitarían ¡45 cables y unos nueve conectores por ordenador!</p>
```

Reply:
Pero la red del ejemplo, hasta ese párrafo, SÍ se limita a dos ordenadores... De hecho, es a partir de este párrafo que se introduce la idea de ampliarla...

Reply:
Tienes razón, lo edito